src/Compilers/Core/Portable/Diagnostic/LocalizableString.cs

```diff
@@ -39,7 +39,12 @@ public string ToString(IFormatProvider formatProvider)
 
         public static implicit operator LocalizableString(string fixedResource)
         {
-            return FixedLocalizableString.Create(fixedResource);
+            return Create(fixedResource);
+        }
+
+        internal static LocalizableString Create(string fixedResource)
```

Comment:
>internal static LocalizableString Create(string fixedResource) [](start = 8, length = 62)

How many consumers for this method? Do we really need it?

Reply:
Done, removed.